libraries/GCS_MAVLink/MissionItemProtocol_Rally.cpp

```diff
@@ -117,24 +117,24 @@ bool MissionItemProtocol_Rally::get_item_as_mission_item(uint16_t seq,
     }
 
     // Default to relative to home
-    ret_packet.frame = MAV_FRAME_GLOBAL_RELATIVE_ALT;
+    ret_packet.frame = MAV_FRAME_GLOBAL_RELATIVE_ALT_INT;
```

Comment:
We do not distinguish these frames.

i.e. An altitude frame doesn't care whether you're transporting things via integer or float.

The creation of these frames is acknowledged as a mistake.